guides/common/attributes-base.adoc

```diff
@@ -17,11 +17,11 @@
 :RepoRHEL7ServerOptional: rhel-7-server-optional-rpms
 :RepoRHEL7ServerAnsible: rhel-7-server-ansible-2.9-rpms
 // For Beta, change to "Beta". For GA releases, change to, for example, "6.8".
-:RepoRHEL7ServerSatelliteServerProductVersion: rhel-7-server-satellite-6.9-rpms
-:RepoRHEL7ServerSatelliteServerProductVersionPrevious: rhel-7-server-satellite-6.8-rpms
-:RepoRHEL7ServerSatelliteCapsuleProductVersion: rhel-7-server-satellite-capsule-6.9-rpms
-:RepoRHEL7ServerSatelliteToolsProductVersion: rhel-7-server-satellite-tools-6.9-rpms
-:RepoRHEL7ServerSatelliteMaintenanceProductVersion: rhel-7-server-satellite-maintenance-6-rpms
+:RepoRHEL7ServerSatelliteServerProductVersion: rhel-7-server-satellite-6-beta-rpms
+:RepoRHEL7ServerSatelliteServerProductVersionPrevious: rhel-7-server-satellite-6.9-rpms
```

Comment:
This feels like a lot of repetition. We already have 6.9 so does this work?
```suggestion
:RepoRHEL7ServerSatelliteServerProductVersionPrevious: rhel-7-server-satellite-{ProductVersion}-rpms
```

Reply:
Please let's not tackle it in this PR to unblock @ryandeussing. Feel free to file a PR that can improve this, there is definitely room for improvement and I agree.